src/boot/efi/boot.c

```diff
@@ -1403,6 +1412,7 @@ static VOID config_load_defaults(Config *config, EFI_FILE *root_dir) {
                 .editor = TRUE,
                 .auto_entries = TRUE,
                 .auto_firmware = TRUE,
+                .auto_enroll = FALSE,
```

Comment:
i think this should default to on, seems riskless, given that people actually have to drop in keys AND the firmware must be explicitly placed in setup mode.

Reply:
I addressed that point in my latest commit.